ntbs-service/Services/LegacySearchService.cs

```diff
@@ -35,20 +37,30 @@ public class LegacySearchService : ILegacySearchService
             SELECT COUNT(*)
             FROM Notifications n";
 
-        private readonly IAuthorizationService _authorizationService;    
 
         private readonly string connectionString;
+        private readonly IReferenceDataRepository _referenceDataRepository;
+        private readonly IConfiguration _configuration;
+        public IList<Sex> Sexes;
 
-        public LegacySearchService(IConfiguration _configuration, IAuthorizationService authorizationService)
+        public LegacySearchService(IConfiguration configuration, IReferenceDataRepository referenceDataRepository)
         {
-            connectionString = _configuration.GetConnectionString("migration");
-            _authorizationService = authorizationService;
+            _configuration = configuration;

```

Comment:
Rather than storing the entire configuration, perhaps fish out the value you care about here?